subsys/bluetooth/host/conn.c

```diff
@@ -1116,6 +1121,64 @@ uint8_t bt_conn_index(struct bt_conn *conn)
 	return (uint8_t)index;
 }
 
+
+#if defined(CONFIG_NET_BUF_LOG)
+struct net_buf *bt_conn_create_pdu_timeout_debug(struct net_buf_pool *pool,
+						 size_t reserve,
+						 k_timeout_t timeout,
+						 const char *func, int line)
+#else
+struct net_buf *bt_conn_create_pdu_timeout(struct net_buf_pool *pool,
+					   size_t reserve, k_timeout_t timeout)
+#endif
+{
+	struct net_buf *buf;
+
+	/*
+	 * PDU must not be allocated from ISR as we block with 'K_FOREVER'
+	 * during the allocation
+	 */
+	__ASSERT_NO_MSG(!k_is_in_isr());
+
+	if (!pool) {
+		return NULL;
+	}
```

Comment:
The earlier version would default to the `acl_tx_pool`, but since this is also used by ISO, then that doesn't really make sense (and also avoids defining the acl pool in broadcast-only scenarios).

Reply:
Update: Had to return it to it's former "glory" as tests failed, but with a guard instead.